lib/shared/ignore.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @fileoverview Create instance of Ignore validator.
+ * @author gfyoung
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const ignoreFactory = require("ignore");
+
+//------------------------------------------------------------------------------
+// Public Interface
+//------------------------------------------------------------------------------
+
+module.exports = (additionalOptions = {}) => {
+    const ignore = ignoreFactory({
+        allowRelativePaths: true,
```

Comment:
I think it would be better to set `allowRelativePaths: true` directly in the rules where the `ignore` package is currently used. The current behavior of those two rules in regard to relative paths was not intended, so it seems better to avoid accidentally introducing the same in other places where we will use the `ignore` package in the future.

Reply:
> The current behavior of those two rules in regard to relative paths was not intended

I'm not sure I understand.  Was supporting relative paths unintentional?

Reply:
> Was supporting relative paths unintentional?

Yes, I believe it was unintentional in `no-restricted-imports` and `no-restricted-modules`, they were meant to work with bare specifiers. But, relative paths also worked because `ignore` didn't complain and users started using that functionality, so we can't remove it now. What I'm suggesting is to set `allowRelativePaths: true` directly in those two rules instead of enabling it by default in `shared/ignore.js`, because that mode isn't desirable.

Reply:
Gotcha, done.